app/src/editor/shared/components/Sidebar/Header.jsx

```diff
@@ -12,22 +12,38 @@ export type Props = {
     title: string,
 }
 
-const Header = ({ title, onClose }: Props) => (
-    <div className={cx(styles.header)}>
-        <div className={styles.titleRow}>
-            <h3 className={cx(styles.name)}>{title}</h3>
-            <button
-                type="button"
-                onClick={() => onClose()}
-                className={styles.iconButton}
-            >
-                <SvgIcon name="crossHeavy" className={styles.icon} />
-            </button>
-        </div>
-        <div className={styles.appInfo}>
-            Streamr App v2.0.5 Beta
+const Header = ({ title, onClose }: Props) => {
+    const { version, branch } = global.streamr.info()
+    const isMaster = branch === 'master'
+    // version e.g. v2.0.5-926-g7e20dd2eb
+    const [versionNumber, , hash] = version.split('-')
+
+    // hash minus leading 'g', not shown on master
+    const displayHash = isMaster ? '' : hash.slice(1)
```

Comment:
Would it make sense to move this logic creating `displayHash, displayBranch` to `app/src/shared/utils/diagnostics.js` and export it there?

Reply:
Not a bad idea